man/runc-kill.8.md

```diff
@@ -7,9 +7,48 @@
 Where "<container-id>" is the name for the instance of the container and
 "<signal>" is the signal to be sent to the init process.
 
+The mapping between "<signal>" and the real golang signal is:
+
+    "ABRT":   syscall.SIGABRT,
```

Comment:
Do you mean the mapping values? I don't know, they are defined by golang package, why would that be directly related to `signal(7)`?
As for the mapping keys, I think they are defined by runc.